projects/components/src/titled-content/titled-content.component.ts

```diff
@@ -75,3 +99,18 @@ export const enum TitlePosition {
   Header = 'header',
   Footer = 'footer'
 }
+
+export const enum TitleStyle {
```

Comment:
Why do we need to introduce new styles? Could you please share the screenshot of how the new mocks is differnet? We should rather make the mock match than introduce new styles

Reply:
I only left some color/justify properties